### PR TITLE
claims status pin to component library 43

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
This pull request uses yarn workspaces to pin the Claims Status Tool to version 43.0.2 of the component library. This is needed because this application still has a need for version 1 of the `va-file-upload` component. 

Screen Cap:
[Screencast from 11-14-2024 02:42:43 PM.webm](https://github.com/user-attachments/assets/37b1bc28-b371-413c-ab39-926bf292d9f6)

screenshots:
image
image
image
image
image
image
image

### CODE REVIEW

File: src/applications/claims-status/components/claim-files-tab/AddFilesForm.jsx [hunk 1]
```diff
@@ -199,6 +199,7 @@ class AddFilesForm extends React.Component {
             name="fileUpload"
             additionalErrorClass="claims-upload-input-error-message"
             aria-describedby="file-requirements"
+            uswds="false"
```
Comment (pmclaren19) [inline src/applications/claims-status/components/claim-files-tab/AddFilesForm.jsx:202]: In this version the default for `uswds` is false so we dont have to set this here.

File: src/applications/claims-status/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (pmclaren19) [inline src/applications/claims-status/package.json:1]: Might be helpful here to add a comment about why we are changing the component library to this version, incase someone tries to change is, especially now that we have two different teams working in this area!

File: src/applications/claims-status/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "name": "@department-of-veterans-affairs/claims-status",
+    "version": "1.0.0",
+    "description": "Dependencies for claims-status tool. Specifically, component library is pinned to 43.0.2 to ensure v1 of the file input component can still be used while the team works on migrating to mulitple file input. For now, please refrain from bumping the comopnent library version in this dependeny list.",
```
Comment (pmclaren19) [inline src/applications/claims-status/package.json:4]: nit - component is mispelled

**Review by pmclaren19 — DISMISSED** [review]
LGTM!

**Review by pmclaren19 — APPROVED** [review]

**Review by CBonade — APPROVED** [review]

**Pull request merged**